internal/provider/apisix/provider.go

```diff
@@ -41,6 +35,12 @@ import (
 	"github.com/apache/apisix-ingress-controller/internal/provider/common"
 	"github.com/apache/apisix-ingress-controller/internal/types"
 	"github.com/apache/apisix-ingress-controller/internal/utils"
+	"github.com/api7/gopkg/pkg/log"
+	"go.uber.org/zap"
+	networkingv1 "k8s.io/api/networking/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	gatewayv1 "sigs.k8s.io/gateway-api/apis/v1"
+	gatewayv1alpha2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
```

Comment:
Third party code libraries are usually located in the middle.

Reply:
what's the issue in this? The autoformatter I have formats it like tihs. I can rearrange but I don't see any issue in this either.

Reply:
fixed